packages/rspack/etc/api.md

```diff
@@ -4074,14 +4074,14 @@ const infrastructureLogging: z.ZodObject<{
     colors?: boolean | undefined;
     console?: Console | undefined;
     debug?: string | boolean | RegExp | ((args_0: string, ...args_1: unknown[]) => boolean) | (string | RegExp | ((args_0: string, ...args_1: unknown[]) => boolean))[] | undefined;
-    level?: "info" | "error" | "none" | "verbose" | "warn" | "log" | undefined;
+    level?: "info" | "none" | "verbose" | "error" | "warn" | "log" | undefined;
```

Comment:
This type is unchanged.

I'm not sure why the order is changed in the generated api file.